r/tests/testthat/test-dplyr-summarize.R

```diff
@@ -17,7 +17,11 @@
 
 skip_if(on_old_windows())
 
-withr::local_options(list(arrow.summarise.sort = TRUE))
+withr::local_options(list(
+  arrow.summarise.sort = TRUE,
+  rlib_warning_verbosity = "verbose",
+  dplyr.summarise.inform = FALSE
```

Comment:
```suggestion
  # This prevents the warning in `summarize()` about having grouped output without 
  # also specifying what to do with `.groups`
  dplyr.summarise.inform = FALSE
```

It might be nice to have a comment about why we need the `dplyr.summarise.inform=FALSE` here so we remember. The others are more-or-less straightforward, but that one I'm not sure I would remember what causes it + where.